### ISSUE TITLE
when I run Quality control,it has no output file

### ISSUE THREAD

**Issue by HHpetra** [ECCC-CDAS/RClimDex#1]
I load the file successfully. but I Press the Perform Quality Control button, then it pop-up a  window. however it has no output file, only has two empty folders.
![image](https://user-images.githubusercontent.com/28735203/45745471-7eb5d580-bc33-11e8-83c0-85e623d78048.png)

**Comment by cicatrixx** [comment]
I am having a similar issue. But I also get the following error message in the console behind the "Processing QC" window:
```
Error in ParseError("[[:alnum:]_.:/\b\\-]*", as.character(tclvalue(rclimdex.env$entries.file.in)),  : 
  (converted from warning) NAs introduced by coercion
```

**Comment by rodneychan-ec** [comment]
There was an compatibility issue with R 3.5.1, should be fixed in new release (1.9-3) now. Otherwise 1.9-2 have to use R 3.3.3 or lower.
